tests/by-util/test_cksum.rs

```diff
@@ -435,6 +471,45 @@ fn test_all_algorithms_fail_on_folder() {
     }
 }
 
+#[test]
+fn test_binary_file() {
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+
+    at.touch("f");
+
+    scene
+        .ucmd()
+        .arg("--untagged")
+        .arg("-b")
+        .arg("--algorithm=md5")
+        .arg(at.subdir.join("f"))
+        .succeeds()
+        .stdout_contains("d41d8cd98f00b204e9800998ecf8427e *");
+
+    // no "*" in this case
+    scene
+        .ucmd()
+        .arg("--tag")
+        .arg("--untagged")
+        .arg("--binary")
+        .arg("--algorithm=md5")
+        .arg(at.subdir.join("f"))
+        .succeeds()
+        .stdout_contains("d41d8cd98f00b204e9800998ecf8427e  ");
```

Comment:
The test asserts that the empty file `f` suddenly becomes plain-text (i.e. un-binary) when `--tag` is given (and later overridden).

However, GNU 9.5 shows no such behavior:

```console
$ ../gnu/src/cksum --untagged --binary --algorithm=md5 emptyfile 
d41d8cd98f00b204e9800998ecf8427e *emptyfile
$ ../gnu/src/cksum --tag --untagged --binary --algorithm=md5 emptyfile 
d41d8cd98f00b204e9800998ecf8427e *emptyfile
```